eduNEXT/edunext-platform
# chore: upgrade Django to 2.2.25

## Description

Upgrade django version from 2.2.20 to 2.2.25

### Other info
For something weird the increasing the version of django was giving webpack error, however since the platform was switched to asg this change works.

## How to test
Enter in lms and studio (in stage) and see that nothing breaks.
Examples: sign in, register, course enrollment, and search course in studio.
